internal/util/slices.go

```diff
@@ -39,3 +41,13 @@ func Equivalent[T comparable](x, y []T) bool {
 
 	return true
 }
+
+// SplitTrim split string and trim whitespace
+func SplitTrim(str, sep string) []string {
```

Comment:
SplitTrim returns a slice with len 1 when str is empty string. not sure if we call this when str is empty string, if there's a chance, i'd suggest to check that. 
https://go.dev/play/p/y50Q0zPqPUm

Reply:
Great observation! Fixed :pray: